<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>581</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Presenter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/icon.svg</normaloff>:/images/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="monitorBox">
      <property name="title">
       <string>Monitor</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
       <item>
        <widget class="QComboBox" name="monitorCombo"/>
       </item>
       <item>
        <widget class="QPushButton" name="monitorRefresh">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="countdownBox">
      <property name="title">
       <string>Countdowns</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QPushButton" name="countdownClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="6">
        <widget class="QListWidget" name="countdownList"/>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="countdownAdd">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="countdownStop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="countdownTest">
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="countdownRemove">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="countdownStart">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="imagesBox">
      <property name="title">
       <string>Images</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,1,1,1,1">
       <item row="0" column="0" colspan="4">
        <widget class="QListWidget" name="imagesList">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideMiddle</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="imagesAdd">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="imagesRemove">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="imagesClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="imagesShow">
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="imagesHide">
         <property name="text">
          <string>Hide</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QFrame" name="imagesPreviewFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="programBox">
      <property name="title">
       <string>Program</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="programStartMinimized">
         <property name="text">
          <string>Start minimized</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="programRunAtStartup">
         <property name="text">
          <string>Run at startup (unimplemented)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="programWarnOnClose">
         <property name="text">
          <string>Warn on close to tray</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>518</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpAboutPresenter"/>
    <addaction name="actionHelpAboutQt"/>
    <addaction name="separator"/>
    <addaction name="actionHelpExit"/>
   </widget>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionHelpAboutPresenter">
   <property name="text">
    <string>&amp;About Presenter...</string>
   </property>
  </action>
  <action name="actionHelpAboutQt">
   <property name="text">
    <string>About &amp;Qt...</string>
   </property>
  </action>
  <action name="actionHelpExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
